### ISSUE TITLE
0.3.0 distribution, at89overlord fails with exception on case-sensitive file systems

### ISSUE THREAD

**Issue by tardate** [piotrb5e3/AT89C2051_programmer#2]
Hi @piotrb5e3, thanks for your project - it was a great help for me building something similar, but with an integrated charge pump - you can see [my project details here](https://github.com/tardate/LittleArduinoProjects/tree/master/8051/AT89C2051/Programmer) if you are interested.

I was able to keep it compatible with at89overlord, and that's the host programming software I'm using.

I ran into one issue after installing the egg under python3. It would complain about a missing `at89overlord.at89comm` module:

```
$ at89overlord 
Traceback (most recent call last):
  File "....Programmer/venv/bin/at89overlord", line 7, in <module>
    from at89overlord.at89runner import main
  File "...Programmer/venv/lib/python3.5/site-packages/at89overlord/at89runner.py", line 4, in <module>
    from at89overlord.at89comm import AT89comm
ImportError: No module named 'at89overlord.at89comm'
```
There appears to be nothing wrong with the source files here, and the `at89overlord-0.3.0.tar.gz` appears ok at https://pypi.org/project/at89overlord/#files 

The problem appears to be in the `at89overlord-0.3.0-py3-none-any.whl` zip that has been distributed. Specifically in `RECORD`, where I see there's been some case confusion, possible from when files were renamed and reorganised on a file system that wasn't fully case-sensitive:
```
at89overlord/AT89comm.py,sha256=AkId-9ahgtrg6ySuNmrt2Em8__jwBb9UTTYWf5knYqo,2053
at89overlord/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
at89overlord/at89comm.py,sha256=T4bd3zeMiPxSHBeJd-uPXXoi-ts8P82oAr2eLFGkRHA,4590
...
```
When I look at the files installed in site-packages, I see I have `AT89comm.py`, not `at89comm.py`, and since my file system is case-sensitive - boom!

I was able to work around this simply by patching the installed file names:
```
mv venv/lib/python3.5/site-packages/at89overlord/AT89comm.py venv/lib/python3.5/site-packages/at89overlord/at89comm.py
```

So long story short - looks like the 0.3.0 distribution is iffy, and would be worth re-packaging an update at some point.

**Comment by piotrb5e3** [comment]
Thank you for the report. I'm looking into it.

**Comment by piotrb5e3** [comment]
I've released the 0.4.0 version. It should fix this problem.

**Comment by tardate** [comment]
confirmed! thanks, 0.4.0 installs and runs just fine.
